.changeset/five-berries-clean.md

```diff
@@ -0,0 +1,5 @@
+---
+'@team-plain/typescript-sdk': major
+---
+
+Changed return type of upsert customer to include the upsert 'result' (e.g. if it the customer was created, updated or not modified). This means that the if you were previously accessing a property of the customer's such as id by doing: `result.id` you now have to change that to be `result.customer.id`.
```

Comment:
I found `result.id` part here confusing, maybe `data.id` or `response.id`would be clearer (as you could have `data/response.result` now after this change

Reply:
Added a code example to this to make it clearer!